engine/src/test/java/io/camunda/zeebe/engine/processing/bpmn/multiinstance/MultiInstanceActivityTest.java

```diff
@@ -363,6 +363,63 @@ public void shouldCompleteBodyWhenLastInstanceCompleteConditionEvaluateTrue() {
             tuple(BpmnElementType.MULTI_INSTANCE_BODY, ProcessInstanceIntent.ELEMENT_COMPLETED));
   }
 
+  @Test
+  public void shouldCompleteBodyWhenCompleteConditionAccessInputDataItemEvaluateTrue() {
+    // given
+    ENGINE
+        .deployment()
+        .withXmlResource(process(miBuilder.andThen(m -> m.completionCondition("= item = 10"))))
```

Comment:
🔧 This test case could be a nice opportunity to test the completion condition evaluating to true once the second task has completed. 

```suggestion
        .withXmlResource(process(miBuilder.andThen(m -> m.completionCondition("= item = 20"))))
```